Sources/FluentBenchmark/Tests/SiblingsTests.swift

```diff
@@ -101,4 +101,26 @@ extension FluentBenchmarker {
             XCTAssertEqual(earth.$tags.pivots.count, 2)
         }
     }
+    
+    private func testSiblings_detachAll() throws {
```

Comment:
You should add a call to this test method in `.testSiblings()`.

Reply:
Ah, totally forgot, thanks!